docs/tutorials/php/build-a-recurring-billing-app/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-id: subscription-tutorial
+id: build-a-recurring-billing-app-php
+title: Build a recurring billing subscription system with PHP
 sidebar_position: 1
+description: In this tutorial you'll build a realistic monthly subscription payments workflow that can be canceled while it runs.
```

Comment:
I don't think it matters but Workflow.

Reply:
I think here we leave it as generic workflow because this is SEO description, not product feature.